downstream/titles/automation-dashboard/automation-dashboard/assembly-view-key-metrics.adoc

```diff
@@ -8,7 +8,7 @@ ifdef::context[:parent-context: {context}]
 
 The {AutomationDashboardName} utility is a web-based container application that provides key metrics related to job execution, efficiency, and the value derived from automation. {AutomationDashboardName} uses automation metrics to provide automation usage data from {PlatformNameShort}. You can use this data to determine the cost of performing tasks manually versus the cost of performing tasks through automation. This comparison is used to demonstrate the savings achievable through automation.
 
-{AutomationDashboardName} helps you to:
+{AutomationDashboardName} designed to help you:
```

Comment:
Hey Kyla! Just a small edit here, "{AutomationDashboardName} is designed to help you:"